api/src/main/java/org/eclipse/microprofile/reactive/messaging/Message.java

```diff
@@ -35,217 +35,6 @@
  */
 public interface Message<T> {
```

Comment:
Ok, this definitely need more discussion because it's it completely derailing with the spec. I doubt we can pass the TCK with such changes.